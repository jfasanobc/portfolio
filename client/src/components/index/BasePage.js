import React, { Component } from 'react';
import HeaderBlock from '../common/header';
import LogoItem from '../common/logo';

export default class BasePage extends Component {
  render() {
    return (
      <div className="grid">
        <HeaderBlock />
        <div className="main">
          <header className="header row">
            <p>Suh Dude!
              <span>@JohnFasano</span>
            </p>
          </header>
          <section className="content row">
            <LogoItem />
          </section>
        </div>
      </div>
    )
  }
}