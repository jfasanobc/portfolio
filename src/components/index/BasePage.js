import React from 'react';
import HeaderBlock from '../common/header';

export default class BasePage extends React.Component {
    render() {
        return (
            <div className="main-body">
                <HeaderBlock />
                <p>Suh Dude! Made Changes</p>
            </div>
        )
    }
}