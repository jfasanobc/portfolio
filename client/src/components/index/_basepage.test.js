import React from "react";
import renderer, { create } from "react-test-renderer";
import BasePage from "./BasePage";

describe("Does the full index/base page load? ", () => {
  test("Loaded in comparison to snapshot", () => {
    const index = create(<BasePage />);
    expect(index.toJSON()).toMatchSnapshot();
  });
});
