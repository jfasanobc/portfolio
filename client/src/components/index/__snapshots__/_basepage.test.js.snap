// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Does the full index/base page load?  Loaded in comparison to snapshot 1`] = `
<div
  className="grid"
>
  <div
    className="grid"
  >
    <div
      className="expanding_nav"
    >
      <img
        alt="Open Menu"
        className="menu_icon center"
        src="./assets/img/menu.png"
      />
      <ul
        className="nav"
      >
        <li
          className="menu_item"
          title="Home"
        >
          <a
            alt="Home"
            href="#"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="menu_item"
          title="About Me"
        >
          <a
            alt="About Me"
            href="#about-me"
            onClick={[Function]}
          >
            About Me
          </a>
        </li>
        <li
          className="menu_item"
          title="Contact Me"
        >
          <a
            alt="Contact Me"
            href="#contact-me"
            onClick={[Function]}
          >
            Contact Me
          </a>
        </li>
        <li
          className="menu_item"
          title="Projects"
        >
          <a
            alt="Projects"
            href="#projects"
            onClick={[Function]}
          >
            Projects
          </a>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="main"
  >
    <header
      className="header row"
    >
      <p>
        Suh Dude!
        <span>
          @JohnFasano
        </span>
      </p>
    </header>
    <section
      className="content row"
    >
      <a
        href="/"
      >
        <img
          className="logo"
          src="./assets/img/logo.png"
        />
      </a>
      <p>
        This is the base content block.
      </p>
    </section>
  </div>
</div>
`;
